src/SDK/MonitoringApplianceEmulator/ViewModels/AlertDetailsControlViewModel.cs

```diff
@@ -29,7 +29,7 @@ public class AlertDetailsControlViewModel : ObservableObject
         private readonly ISystemProcessClient systemProcessClient;
 
         // temporary, until all types will be supported
-        private readonly List<AlertPropertyType> supportedPropertiesTypes = new List<AlertPropertyType>() { AlertPropertyType.Text, AlertPropertyType.KeyValue, AlertPropertyType.Table };
+        private readonly List<AlertPropertyType> supportedPropertiesTypes = new List<AlertPropertyType>() { AlertPropertyType.Text, AlertPropertyType.KeyValue, AlertPropertyType.Table, AlertPropertyType.Chart };
```

Comment:
>supportedPropertiesTypes [](start = 49, length = 24)

what's not supported right now?

Reply:
full text

---
In reply to: [231015720](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/17#discussion_r231015720) [](ancestors = 231015720)